spk/tt-rss/Makefile

```diff
@@ -12,7 +12,10 @@ DESCRIPTION_FRE = Tiny Tiny RSS est un agrégateur et lecteur web de flux RSS et
 ADMIN_URL = /tt-rss/
 RELOAD_UI = yes
 DISPLAY_NAME = Tiny Tiny RSS
-CHANGELOG = "Update Tiny Tiny RSS to Git revision 67f02e2aa7 (20200829)"
+CHANGELOG = "
+CHANGELOG += 13. Update Tiny Tiny RSS to Git revision 328d7b55c8 (20201114), add troubleshooting logs generation & fixed rights issue with upgrading on a live installation<br />
+CHANGELOG += 12. Update Tiny Tiny RSS to Git revision 67f02e2aa7 (20200829)<br />
+CHANGELOG += "
```

Comment:
The changelog numbering is proposed to start at "1." within each SPK_REV.
And it should contain entries for the dedicated SPK_REV only.
So you can ommit the numbering and the entry starting with 12.